docs/smart-contracts/getting-started/setup.mdx

```diff
@@ -105,29 +105,28 @@ soroban
 $ soroban
 Build, deploy, & interact with contracts; set identities to sign with; configure networks; generate keys; and more.
 
-Stellar Docs: https://developers.stellar.org
-CLI Full Hep Docs: https://github.com/stellar/soroban-cli/tree/main/FULL_HELP_DOCS.md
+Intro: https://soroban.stellar.org/docs
+CLI Reference: https://github.com/stellar/soroban-cli/tree/main/docs/soroban-cli-full-docs.md
 
 Usage: soroban [OPTIONS] <COMMAND>
 
 Commands:
   completion  Print shell completion code for the specified shell
+  config      Deprecated, use `soroban keys` and `soroban network` instead
   contract    Tools for smart contract developers
   events      Watch the network for contract events
   keys        Create and manage identities including keys and addresses
-  xdr         Decode and encode XDR
```

Comment:
This wasn't removed.

Reply:
this is exactly what the cli is showing (`soroban`). Can you please confirm the output on your end?